packages/sessions-sdk-react/src/session-limits.tsx

```diff
@@ -206,7 +227,19 @@ const Token = ({
 
     case StateType.Loading:
     case StateType.NotLoaded: {
-      return "Loading..."; // TODO
+      return (
+        <TextField
```

Comment:
what's the reason for returning a `TextField` with `isPending` here instead of a simpler component?